### ISSUE TITLE
Trusted Tester comment on WCAG3 draft: Conformance Levels

### ISSUE THREAD

**Issue by kengdoj** [w3c/wcag3#254]
Thank you for the opportunity to comment. Apologies for the delayed response. We hope it's not too late and is still useful.

## Conformance Levels

Code-based testing and AT testing, while both valuable, can yield very different results. How WCAG 3 will combine these results is unclear, and we have concerns that combining results will result in loss of clarity with regard to actual conformance to the defined standards/guidelines.

Code-based evaluation (the current most common way of testing) is a determination of how well content meets the guidelines, and the results will be as subjective as the guidelines. If content includes the properties required to meet the guidelines, it passes.

AT testing is dependent on a 3rd party tool&#39;s compatibility with the code, the tool vendor&#39;s interpretation of accessibility guidelines and technology specifications, as well as the tool&#39;s presumed and designed intent to help users navigate accessibility challenges. The purpose of assistive technology is to assist and provide accessibility; therefore, AT testing results are not always a good indication of the content&#39;s conformance to the guidelines. AT testing indicates the ability of the AT to provide information from the content, not necessarily whether the content itself conforms to accessibility guidelines/standards.

The conformance levels bronze, silver, gold, with code-based inspection representing the minimum (bronze) level of testing implies that AT testing is actually better than code-based inspection. This is not a concept that we support. Similarly, the use of the term &quot;holistic tests&quot; to describe testing with AT implies that AT testing is more comprehensive, and therefore better at determining accessibility. While we encourage AT testing as a tool in usability testing and to understand the experience of AT users, for &quot;conformance testing&quot;, we strongly believe that content and code evaluation is more accurate and ensures broader support for all assistive technologies. When testing with a particular AT, some test results are only indicative of a user&#39;s experience using that specific AT, with limited external validity for some results. Code-based inspection, on other hand, provides a more accurate representation of conformance to guidelines and standards and is typically more broadly indicative of accessibility, regardless of the specific AT a user may employ.

In the early years of Trusted Tester (around 2008), the JAWS screen reader was one of the testing tools incorporated in the test process, in addition to code evaluation tools. We taught use of JAWS in Trusted Tester training classes to student testers who were introduced to assistive technology in the same class. We found that JAWS was difficult to teach, and it was even more difficult to ensure that testers who left our training were able to use JAWS at a satisfactory level. Within the testing team at DHS&#39;s Office of Accessible Systems &amp; Technology, the creator of Trusted Tester, we found conflicting results between code evaluation results and JAWS testing. The most common scenario was that JAWS was not reading content that passed code evaluation (code passed but JAWS failed), but there were also instances where JAWS compensated for poorly coded content (code failed but JAWS passed). In all situations, the code evaluation result overruled the JAWS result because another screen reader confirmed the code evaluation result. In another instance, when our JAWS version was updated, we found that the updated version of JAWS did not read the same content that was read successfully with the previous version of JAWS. After a year of testing with JAWS and JAWS results being overruled or inconsistent, JAWS was dropped from Trusted Tester as a testing tool.

As such, AT testing is not &quot;better&quot;, nor does it improve code evaluation results. The results may conflict, as described in the previous paragraph. AT testing provides valuable but different information – about the AT. Summarizing, AT testing can answer questions like &quot;Does this AT (specific product and version) work well with this application?&quot;, while code evaluation answers the question &quot;does this application meet accessibility requirements?&quot;

**Comment by hidde** [comment]
thank you for the feedback. A new conformance section and a reporting section with levels has been published in [the latest working draft](https://www.w3.org/TR/wcag-3.0/), and has accessibility support sets specifically as part of the conformance section. As they address the issues described here, I am closing this issue. 

We welcome feedback on the latest working draft.
